src/mobu/handlers/external.py

```diff
@@ -104,6 +111,20 @@ async def get_flock(
     return context.manager.get_flock(flock).dump()
 
 
+@external_router.put(
+    "/flocks/{flock}",
+    responses={404: {"description": "Flock not found", "model": ErrorModel}},
+    status_code=202,
+    summary="Signal a flock to refresh",
+)
+async def refresh_flock(
+    flock: str,
+    context: Annotated[RequestContext, Depends(context_dependency)],
+) -> None:
+    context.manager.refresh_flock(flock)
+    context.logger.info("Signaling flock to refresh", flock=flock)
```

Comment:
Minor thing, but I prefer to log before the action rather than afterwards so that you get the log message if the action blows up with an exception. It helps make sense of the exception.